status/src/main/resources/rp-tooling.conf

```diff
@@ -0,0 +1,9 @@
+akka {
+  extensions += "com.lightbend.rp.status.ApplicationStatus"
```

Comment:
You should rather set the: 

```
  # List FQCN of extensions which shall be loaded at actor system startup.
  # Library extensions are regular extensions that are loaded at startup and are
  # available for third party library authors to enable auto-loading of extensions when
  # present on the classpath. This is done by appending entries:
  # 'library-extensions += "Extension"' in the library `reference.conf`.
  #
  # Should not be set by end user applications in 'application.conf', use the extensions property for that
  #
  library-extensions = ${?akka.library-extensions} []
```

Reply:
it can be done in just reference.conf, not sure why the new conf file is needed?

Reply:
Now using `akka.library-extensions`.